.github/workflows/desktop_make.yml

```diff
@@ -15,20 +15,8 @@ jobs:
       - name: Package app
         run: ./gradlew copyBrandingToCommonResources package -Porganization=ooni
```

Comment:
Now this will only build for the current platform right? We need to run this on 2 or 3 different machines to get our binaries.